### PR TITLE
Feature/checkbox cleanup

### PR DESCRIPTION
**Pull request by jrauh**
- [x] Desktop - Chrome
- [x] Desktop - Firefox
- [x] Desktop - Safari
- [x] Desktop - Edge
- [x] Desktop - IE 11 (partial support with fallbacks – text does not indent properly)
- [x] iOS - Safari
- [x] iOS - Chrome
- [x] Android - Chrome

### CODE REVIEW

**Comment by jrauh** [comment]
The only issue I've noticed with this is if Source Sans doesn't load. The fallback font (helvetica) is a different size, which affects the positioning of the `•` inside the radio button. A fix could be to create a circle as a dashing-icon.

File: sass/modules/forms/themed-elements.scss [hunk 1]
```diff
@@ -167,51 +167,36 @@ $switch--off: $gray-250 !default; //Color of switch when off
...
-      content: '';
-      border: 2px solid $gray-200;
+    // position: relative;
+
+    //Radio Button Styles
+    &:before {
+      content: '•';
       border-radius: 50px;
-      display: block;
       font-size: 1.7rem;
```
Comment (aknobloch3) [inline sass/modules/forms/themed-elements.scss]: Is it just me, or does the dot look a little off on Tablet view with this `font-size`?

**Comment by aknobloch3** [comment]
Also, I think the Default Radio lost its hover effect of showing the dot on hover?

File: sass/modules/forms/themed-elements.scss (hunk 1 above)
Comment (jrauh) [reply sass/modules/forms/themed-elements.scss]: It's not you. There was a media query targeting desktop. It should have been tablet. I'll fix it

**Pull request merged**